UrbanMobility/StopTime/doc/spec.md

```diff
@@ -0,0 +1,103 @@
+# gtfs:StopTime
+
+## Description
+
+See [https://developers.google.com/transit/gtfs/reference/#stop_timestxt](https://developers.google.com/transit/gtfs/reference/#stop_timestxt)
+
+## Data Model
+
++ `id`: Entity id. 
+    + It shall be `urn:ngsi-ld:gtfs:StopTime:<stop_time_identifier>` being `stop_time_identifier` a value that can be derived from GTFS `trip_id` and `stop_id`. 
+
++ `type`: Entity type. 
+    + It shall be equal to `gtfs:StopTime`.
+    
++ `dateCreated` : Entity's creation timestamp.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated. 
+ 
++ `dateModified`: Last update timestamp of this Entity.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated.
+  
++ `hasTrip`: Same as GTFS `trip_id`.
+  + Attribute type: Relationship. It shall point to an Entity of type [gtfs:Trip](../../../Trip/doc/spec.md)
+  + Mandatory
+
++ `hasStop`: Same as GTFS `stop_id`
+  + Attribute type: Relationship. It shall point to an Entity of type [gtfs:Stop](../../../Stop/doc/spec.md)
+  + Mandatory
+
++ `arrivalTime`: Same as GTFS `arrival_time`
+  + Attribute type: Property. [Text](https://schema.org/Text).
```

Comment:
why not a proper data type?

Reply:
several reasons:

* because Orion Broker does not support a Time data type
* because Google use their own conventions and I don't want to change them.

Reply:
sure, but still, proper string formatting helps the semantics (btw i think linked gtfs datatype may be wrong...) it's set as xsd:duration but that's not correct afaik